inginious/frontend/user_manager.py

```diff
@@ -107,6 +115,12 @@ def sanitize_email(cls, email: str) -> str:
     #           User session management          #
     ##############################################
 
+    def get_theme(self):
```

Comment:
I'd rather call them `session_theme` and `session_codemirror_theme` to use the same nomenclature and recall throughout the code that those values are fetched from the user session.